doc/CONFIGURATION.json

```diff
@@ -1629,6 +1629,20 @@
     "status" : "advanced",
     "type" : "bool"
   },
+  {
+    "autoExpansion" : "none",
+    "category" : "Passcode",
+    "categoryTag" : "passcode",
+    "classIdentifier" : "passcode",
+    "className" : "ownCloudAppShared.AppLockManager",
+    "defaultValue" : 4,
+    "description" : "Controls how many passcode digits are minimal required for passcode lock.",
```

Comment:
@hosy I find wording "digits minimal required" a bit strange. Eventually "at least required"?

Reply:
Thanks, wording is fixed now.